app/config.py

```diff
@@ -146,6 +147,15 @@ class Config:
             "options": "-c statement_timeout=1200000",
```

Comment:
Is it worth also removing this? I know the `SET statement_timeout` in `connect()` will overwrite it but it seems confusing to configure the same thing in two places

Reply:
Indeed perhaps I should remove it... I'm just a little unsure whether its replacement gets run for things like our alembic migrator.